runtime/criusupport/criusupport.cpp

```diff
@@ -443,6 +464,7 @@ Java_org_eclipse_openj9_criu_CRIUSupport_checkpointJVMImpl(JNIEnv *env,
 		char *syslogOptions = NULL;
 		I_32 syslogBufferSize = 0;
 		UDATA oldVMState = VM_VMHelpers::setVMState(currentThread, J9VMSTATE_CRIU_SUPPORT_CHECKPOINT_PHASE_START);
+		BOOLEAN isJdwpEnabled = FIND_ARG_IN_VMARGS(STARTSWITH_MATCH, "-agentlib:jdwp", NULL) >= 0;
```

Comment:
add a state variable to `J9CRIUCheckpointState` and do the check on `setupJNIFieldIDsAndCRIUAPI` instead

Reply:
Added in https://github.com/eclipse-openj9/openj9/pull/16653/commits/15a2fc49467422d926e8a58659658e35876c4e14